apps/chat2/chat2.nim

```diff
@@ -38,7 +38,7 @@ when defined(rln):
   import
     libp2p/protocols/pubsub/rpc/messages,
     libp2p/protocols/pubsub/pubsub,
-    ../../waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils
+    ../../waku/v2/protocol/waku_rln_relay/utils
```

Comment:
Here we should also import the barrel import module

Reply:
Addressed in https://github.com/status-im/nwaku/commit/e9d15c538c906cba1b2a87d5fa31d301e991e2df